link/api/link.api

```diff
@@ -68,6 +76,15 @@ public final class com/stripe/android/link/ui/ComposableSingletons$LinkButtonKt
 	public final fun getLambda-5$link_release ()Lkotlin/jvm/functions/Function2;
 }
 
+public final class com/stripe/android/link/ui/ComposableSingletons$LinkTermsKt {
```

Comment:
Do these go away if we `@file:Restrict` them?

Reply:
Doesn’t look like it 😔